packages/docs/pages/radio.tsx

```diff
@@ -1,151 +1,219 @@
-                        disabled={true}
-                        checked={selected === 'disabled'}
-                        value="disabled"
-                        onChange={handleChange}
-                      />
-                    </FormGroup>
-                  </Form>
-                );
-              }}
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-          <Panel header="Grouping">
-            <Text>
-              In order to group radio controls, use the <Code>Fieldset</Code> component to separate the controls.
-            </Text>
+      <Panel header="Implementation" headerId="implementation">
+        <ContentRoutingTabs
+          id="implementation"
+          routes={[
+            {
+              id: 'basic',
+              title: 'Basic',
+              render: () => (
+                <Fragment key="basic">
+                  <Text>
+                    <Code primary>Radios</Code> are group of items which a single option can be selected.
+                  </Text>
+                  <CodePreview>
+                    {/* jsx-to-string:start */}
+                    {function Example() {
+                      const [selected, setSelected] = useState('');
 
-            <CodePreview>
-              {/* jsx-to-string:start */}
-              {function Example() {
-                const [firstRadio, setFirstRadio] = useState('');
-                const [secondRadio, setSecondRadio] = useState('');
+                      const handleChange = (event) => setSelected(event.target.value);
 
-                const handleFirstChange = (event) => setFirstRadio(event.target.value);
-                const handleSecondChange = (event) => setSecondRadio(event.target.value);
+                      return (
+                        <Form>
+                          <FormGroup>
+                            <Radio label="On" checked={selected === 'on'} value="on" onChange={handleChange} />
+                            <Radio label="Off" checked={selected === 'off'} value="off" onChange={handleChange} />
+                            <Radio
+                              label="Disabled"
+                              disabled={true}
+                              checked={selected === 'disabled'}
+                              value="disabled"
+                              onChange={handleChange}
+                            />
+                          </FormGroup>
+                        </Form>
+                      );
+                    }}
+                    {/* jsx-to-string:end */}
+                  </CodePreview>
+                </Fragment>
+              ),
+            },
+            {
+              id: 'grouping',
+              title: 'Grouping',
+              render: () => (
+                <Fragment key="grouping">
+                  <Text>
+                    In order to group radio controls, use the <Code>Fieldset</Code> component to separate the controls.
```

Comment:
Tried to find the `Fieldset` page, but I didn't find it, Is that expected?
Should  we remove the `<Code/>` tag?

Reply:
I guess we're referencing the html element. I would remove the `Code` tag.

Reply:
We have a component for [`Fieldset`](https://github.com/bigcommerce/big-design/blob/master/packages/big-design/src/components/Fieldset/Fieldset.tsx) but we don't have a dedicated page for it. It's located on the `Forms` page props.

Reply:
We group `Fieldset` in the `Form` component. I think it's fine to leave as is.